holoviews/core/data/__init__.py

```diff
@@ -173,7 +173,7 @@ class Dataset(Element):
 
     # In the 1D case the interfaces should not automatically add x-values
     # to supplied data
-    _auto_indexable_1d = False
+    _auto_indexable_1d = True
```

Comment:
Is this correct? The auto indexing behavior caused problems fixed in a recently merged PR...